plugins/language_cmake.lua

```diff
@@ -4,14 +4,17 @@ local syntax = require "core.syntax"
 syntax.add {
   name = "CMake",
   files = { "%.cmake$", "CMakeLists.txt$" },
-  comment = "//",
+  comment = "#",
+  block_comment = { "#[[", "]]" },
   patterns = {
-    { pattern = { "#", "[^\\]\n" },     type = "comment"  },
-    { pattern = { '"', '"', '\\' },     type = "string"   },
-    { pattern = { "'", "'", '\\' },     type = "string"   },
-    { pattern = "[%a_][%w_]*%f[(]",     type = "function" },
-    { pattern = "[%a_][%w_]*",          type = "normal"   },
-	{ pattern = "%${[%a_][%w_]*%}",     type = "operator"   },
+    { pattern = { "#%[%[", "%]%]"},      type = "comment"  },
+    { pattern = { '#%[=*%[', '%]=*%]' }, type = "comment"  },
```

Comment:
I think the first one isn't needed, as the second one covers it.

Reply:
true, the * covers from 0 to anyone, I forgot...